bgp/modules/terms.py

```diff
@@ -61,18 +65,56 @@ def results(self):
 class ReadingLevelModule:
 
     def __init__(self):
-        self.flesch_kincaid_grade= None
+#        self.flesch_kincaid_grade= None
+        self.isbn= None
+        self.lexile_min_age= 'None'
+        self.lexile_max_age= 'None'
+        self.readability_fk_score= None
+        self.readability_s_score= None
         self.time = 0
 
     def run(self, doc, **kwargs):
-        import textstat
-        self.flesch_kincaid_grade = textstat.flesch_kincaid_grade(doc)
+        import requests
+        from readability import Readability
+        from readability.scorers.flesch_kincaid import ReadabilityException
```

Comment:
We may want to eventually move these dependencies (especially `requests`) outside of this method

see PEP8:
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
This is addressed in Move dependencies to top of file #48